### PR TITLE
Refactor some code

### PR DESCRIPTION
**Pull request by FrankFang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/button/button.tsx [hunk 1]
```diff
@@ -222,19 +220,8 @@ const InternalButton: React.ForwardRefRenderFunction<unknown, ButtonProps> = (pr
...
-    case 'large':
-      sizeCls = 'lg';
-      break;
-    case 'small':
-      sizeCls = 'sm';
-      break;
-    default:
-      break;
-  }
+  const abbrs = { large: 'lg', small: 'sm' };
```
Comment (afc163) [inline components/button/button.tsx:223]:
```suggestion
  const sizeClassNameMap = { large: 'lg', small: 'sm' };
```

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/runs/4327100993?check_suite_focus=true

**Comment by afc163** [comment]
ping

**Comment by FrankFang** [comment]
@afc163  fixed

**Pull request merged**